proc_nutation.py

```diff
@@ -37,7 +37,8 @@
     # {{{ do the centering before anything else!
     # in particular -- if you don't do this before convolution, the
     # convolution doesn't work properly!
-    else:
+    s.ift('t2')
+    if 'amp' not in s.dimlabels:
```

Comment:
this is what I was referring to as the "hanging else" -- it's the same level of indentation with the previous line -- it's not paired with an if statement.

It actually *is* a syntax error, but you don't see it b/c you quit previous to it!